frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailsTable.tsx

```diff
@@ -73,10 +74,13 @@ export function DetailsTableCell({
       const formattedJson = JSON.stringify(value, null, 2);
       cellValue = <pre className="ObjectJSON">{formattedJson}</pre>;
     } else {
+      const tableColumnSettings =
```

Comment:
a side effect here is that other display formats like link/email/etc would also carry over to this details table... I suspect that could be good..? but also super unsure heh.